etna/pipeline/base.py

```diff
@@ -456,6 +456,7 @@ def _get_backtest_forecasts(self) -> pd.DataFrame:
             forecast = forecast.join(fold_number_df)
             forecasts_list.append(forecast)
         forecasts = pd.concat(forecasts_list)
+        forecasts.sort_index(axis="columns", ascending=[True, False], inplace=True)
         return forecasts
```

Comment:
We dont need specify `ascending` order here.

Reply:
Checks failed with the following error. I cannot understand, what should I do
`
ImportError while loading conftest '/home/runner/work/etna/etna/tests/test_ensembles/conftest.py'.
tests/test_ensembles/conftest.py:16: in <module>
    from etna.models import ProphetModel
E   ImportError: cannot import name 'ProphetModel' from 'etna.models' (/home/runner/work/etna/etna/etna/models/__init__.py)
Error: Process completed with exit code 4.
`

Reply:
It's ok, will fix it today or next day

You can test locally, via command `poetry run pytest tests -v --cov=etna -m "not long" --cov-report=xml` and run linters via `make lint`

Reply:
@mvakhmenin  We've merged updates to master branch. You can rebase on updated `master`.

Reply:
@martins0n 
Tests fail without ascending order

Reply:
@mvakhmenin  We don't expect such ordering, it's better to change fixtures in backtest to make tests pass

Reply:
@martins0n 
I do not understand, what's the problem with lint